src/components/TabQueryEditor.vue

```diff
@@ -202,6 +203,13 @@
         const rows = this.result.affectedRows || 0
         return `${rows} ${Pluralize('row', rows)} affected`
       },
+      executeTimeText() {
+        if (!this.result) {
+          return null
+        }
+        const executeTime = this.result.executeTime || 0
+        return `${executeTime} ms`
```

Comment:
We could use this simple package to humanize the time - https://github.com/EvanHahn/HumanizeDuration.js

Reply:
Good point, I have addressed that now